publish-docs

```diff
@@ -35,8 +35,3 @@ cd ..
 
 # publish the docs
 # you will need push access to the mongo-java-driver repo's gh-pages branch
-cd mongo-java-driver
-git add .
-git ci -a -m fixup!
-git rebase -i --root
-git push origin gh-pages -f
```

Comment:
Suggestion: Could the push instructions be added to the README if they are removed from the build script? The two main ones I think could be a little tricky are lines 38 (directory to add) and 42 (force push to gh-pages branch).